src/bandersnatch_filter_plugins/blacklist_name.py

```diff
@@ -72,26 +71,55 @@ def check_match(self, **kwargs):
         bool:
             True if it matches, False otherwise.
         """
-        logger.debug('Running filter plugin %s', self.name)
         name = kwargs.get('name', None)
         if not name:
             return False
 
-        logger.info(
-            'Checking for package %s in %r', name, self.blacklist_package_names
-        )
         if name in self.blacklist_package_names:
-            logger.debug(
-                'MATCH: Package %r is in %r', name,
-                self.blacklist_package_names
-            )
+            logger.debug(f'MATCH: Package {name!r} is in the blacklist')
```

Comment:
Isn't name a str, why do you want the __repr__ of it?

Reply:
It put's the string quotes around the name